src/server/util/aws.ts

```diff
@@ -0,0 +1,36 @@
+import { parse } from 'url'
+import { S3 } from 'aws-sdk'
+import { s3Bucket } from '../config'
+
+export const s3 = new S3()
+
+/**
+ * Reformat the pre-signed url to to one that will be accepted by S3.
+ *
+ * This is necessary because we used a gov.sg CNAME and configured S3
+ * to only accept requests that have that gov.sg CNAME as the path.
```

Comment:
could you remind me again on which configuration resulted in this restriction?

Reply:
This happened after we did both the CNAME, SSL cert, and cloudfront configs, so I'm not sure exactly which one caused this behaviour. If I were to guess, it's likely related to [this](https://aws.amazon.com/premiumsupport/knowledge-center/cloudfront-access-to-amazon-s3/)

Reply:
I don’t remember setting that, will investigate